src/services/review-service.js

```diff
@@ -1,10 +1,23 @@
-import request from "./request";
+import request from './request';
 
-class reviewServise{
+class reviewServise {
+  async comment({ userJwt, comment, placeId, rating }) {
+    const res = await request.post('reviews', {
```

Comment:
Please use requests in try\catch block.

Reply:
There was no changes on my part in this code, aside from Prettier automatic code formatting.